### PR TITLE
Restore font and Blaze3D mappings

### PR DESCRIPTION
**Pull request by violet-sh**
It seems quite a few things were lost in the move to JOML.

### CODE REVIEW

**Review by LambdAurora — APPROVED** [review]

File: mappings/com/mojang/blaze3d/platform/GlStateManager.mapping [hunk 1]
```diff
@@ -352,6 +352,16 @@ CLASS com/mojang/blaze3d/platform/GlStateManager com/mojang/blaze3d/platform/GlS
 		ARG 1 strings
 	METHOD lambda$static$0 (I)Lcom/mojang/blaze3d/platform/GlStateManager$C_vgejkzlz;
 		ARG 0 id
+	METHOD setupGui3DDiffuseLighting (Lorg/joml/Vector3f;Lorg/joml/Vector3f;)V
```
Comment (IotaBread) [inline mappings/com/mojang/blaze3d/platform/GlStateManager.mapping:355]: I'm not sure if we use `3D` or `3d`

File: mappings/com/mojang/blaze3d/systems/RenderSystem.mapping [hunk 2]
```diff
@@ -282,17 +289,32 @@ CLASS com/mojang/blaze3d/systems/RenderSystem com/mojang/blaze3d/systems/RenderS
...
 		ARG 0 location
 		ARG 1 texture
+	METHOD setTextureMatrix (Lorg/joml/Matrix4f;)V
+		ARG 0 textureMatrix
 	METHOD setupDefaultState setupDefaultState (IIII)V
 		ARG 0 x
 		ARG 1 y
 		ARG 2 width
 		ARG 3 height
+	METHOD setupGui3DDiffuseLighting (Lorg/joml/Vector3f;Lorg/joml/Vector3f;)V
```
Comment (IotaBread) [inline mappings/com/mojang/blaze3d/systems/RenderSystem.mapping:308]: Same here

File: mappings/net/minecraft/client/render/Frustum.mapping [hunk 3]
```diff
@@ -1,16 +1,30 @@
...
+		ARG 1 modelViewMatrix
+		ARG 2 projectionMatrix
 	METHOD m_fsdaswmf isVisible (Lnet/minecraft/unmapped/C_hbcjzgoe;)Z
 		ARG 1 box
 	METHOD m_fzvuuyma init (Lorg/joml/Matrix4f;Lorg/joml/Matrix4f;)V
+		ARG 1 view
+		ARG 2 projection
 	METHOD m_gthtrhnt offsetToIncludeCamera (I)Lnet/minecraft/unmapped/C_jwzjqevg;
 		ARG 1 offset
+	METHOD m_vzmlhobi isCubeInFrustrum (DDDDDD)Z
```
Comment (IotaBread) [inline mappings/net/minecraft/client/render/Frustum.mapping]:
```suggestion
	METHOD m_vzmlhobi isCubeInFrustum (DDDDDD)Z
```

**Review by IotaBread — APPROVED** [review]

File: mappings/com/mojang/blaze3d/platform/GlStateManager.mapping (hunk 1 above)
Comment (violet-sh) [reply mappings/com/mojang/blaze3d/platform/GlStateManager.mapping:355]: `3D` had been used in 1.19.2 so I carried it over, I can change it if wanted.

Comment (IotaBread) [reply mappings/com/mojang/blaze3d/platform/GlStateManager.mapping:355]: It's alright then

**Review by eristhea — APPROVED** [review]

**Pull request merged**